diff --git a/dist/dev.js b/dist/dev.js
index 36ca7b5680d5b127cfbd2e3fdfd76f3abbc02ad4..112d63f7d3c55bd7e178be6a5e9ed640caa8a617 100644
--- a/dist/dev.js
+++ b/dist/dev.js
@@ -219,7 +219,7 @@ function resolvePluginConfig(config, options) {
   const { reactRouterConfig, environmentBuildContext, rootDirectory } = config.__reactRouterPluginContext;
   const buildDirectory = path.relative(rootDirectory, reactRouterConfig.buildDirectory);
   const appDirectory = path.relative(rootDirectory, reactRouterConfig.appDirectory);
-  const isSsrBuild = environmentBuildContext?.name === "ssr";
+  const isSsrBuild = environmentBuildContext?.name === "ssr" || environmentBuildContext?.name?.startsWith("ssr");
   const assetsDir = config.build?.assetsDir || "assets";
   const serverEntryPoint = options.serverEntryPoint || findDefaultServerEntry(appDirectory);
   const serverBuildFile = reactRouterConfig.serverBuildFile;
